go/libraries/doltcore/doltdb/root_val_test.go

```diff
@@ -98,3 +100,222 @@ func TestTableDiff(t *testing.T) {
 		t.Error("Bad table diff after adding a second table")
 	}
 }
+
+func TestDocDiff(t *testing.T) {
+	ddb, _ := LoadDoltDB(context.Background(), types.Format_7_18, InMemDoltDB)
+	ddb.WriteEmptyRepo(context.Background(), "billy bob", "bigbillieb@fake.horse")
+
+	cs, _ := NewCommitSpec("head", "master")
+	cm, _ := ddb.Resolve(context.Background(), cs)
+
+	root, err := cm.GetRootValue()
+	assert.NoError(t, err)
+
+	docDetails := make([]DocDetails, 2)
+	doc1 := DocDetails{DocPk: LicensePk}
+	doc2 := DocDetails{DocPk: ReadmePk}
+	docDetails[0] = doc1
+	docDetails[1] = doc2
```

Comment:
docDetails := []DocDetails {
        {DocPk: LicensePk}
        {DocPk: ReadmePk}
    }

Reply:
Thanks, using it now: https://github.com/liquidata-inc/dolt/pull/302/files#diff-4574e1ccc92bceb6353727b26c3f2b94R116